news/2018-09-09-project-loved-week-of-september-9th.md

```diff
@@ -0,0 +1,230 @@
+
+[![](/wiki/shared/news/2018-09-09-project-loved-week-of-september-9th/osu/forward.jpg)](https://osu.ppy.sh/community/forums/topics/801918)
+
+[Gohma - Forward](https://osu.ppy.sh/beatmapsets/731899) by [Evgerable](https://osu.ppy.sh/users/5657937)  
+*written by [DigitalHypno](https://osu.ppy.sh/users/4384207)*
+
+This song, Forward, definitely has a jumpy feel to it. Evgerable captured this feeling masterfully in his mapset of 5 different difficulties. The star ratings range from low 5 stars to mid 7 stars, filling every gap in-between, so no matter what skill level you're at, you're sure to find a difficulty you enjoy. The gimmick of this mapset is slightly larger CS than the norm, with most difficulties using CS 3.6 and the bottom difficulty using CS 3.2. If that sounds like your thing, come join the 200+ players who already have this mapset favorited, and be sure to vote it for loved!
+
+[![](/wiki/shared/news/2018-09-09-project-loved-week-of-september-9th/osu/ghost-rule.jpg)](https://osu.ppy.sh/community/forums/topics/801919)
+
+[DECO\*27 - Ghost Rule](https://osu.ppy.sh/beatmapsets/425169) by [val0108](https://osu.ppy.sh/users/243917), [Kenterz](https://osu.ppy.sh/users/1587976), [z1085684963](https://osu.ppy.sh/users/1816779) and [caren\_sk](https://osu.ppy.sh/users/542498)  
+*written by [DigitalHypno](https://osu.ppy.sh/users/4384207)*
+
+A song popularized by a few other mapsets in the Ranked and Loved categories, val0108's mapset of Ghost Rule with 4 different difficulties from 4 different mappers—Kenterz, val0108, z1085684963 and caren\_sk—does a great job capturing the song in styles that aren't offered by the other sets. It certainly shows, too: the mapset has over 190 favorites, and that number is sure to increase. Whether you're a fan of unconventional flow or large, flowy, full screen jumps, this mapset has something for you.
+
+[![](/wiki/shared/news/2018-09-09-project-loved-week-of-september-9th/osu/calling-out-feat-kg-skyelle-205.jpg)](https://osu.ppy.sh/community/forums/topics/801920)
+
+[Muzzy - Calling Out (feat. KG & Skyelle) 205](https://osu.ppy.sh/beatmapsets/392041) by [elchxyrlia](https://osu.ppy.sh/users/1722835)  
+*written by [Toy](https://osu.ppy.sh/users/2757689)*
+
+If you've played one lahphnya map, you've played them all. Right? I'm not here to answer that, but if that's the case, then whatever elchxyrlia is doing, he's been doing it right for years. Yet another nearly 200 favorite Speed Up Ver. of an already exciting fast paced song, mixed with spaced streams and full screen jumps, all set to roast together in the oven and you've got a decked out 8\* mammoth of a map.
```

Comment:
Maybe note that lahphnya was elchxyrlia's former name?

Reply:
think it's implied in the next sentence